### PR TITLE
BE Send notification when add/remove user from group, or approve/reject request change email

### PR DESCRIPTION
**Pull request by phandinhman**
## TODO

BE Send notification when add/remove user from group, or approve/reject request change email
## Related URLs

- Issue: https://link/to/issue

## UI changes

N/A

### CODE REVIEW

File: app/Services/NotificationService.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class NotificationService
+{
+
+    /**
+     * @return object
+     */
+    public function getContentNotification($action, $groupName, $userId)
+    {
+        $message = __('notification.' . $action);
+        $notification = [
```
Comment (quynt-1571) [inline app/Services/NotificationService.php]: `return [
            'message' => $message . $groupName,
            'userId' => $userId,
            'read' => false
        ];`

File: app/Events/UserNotification.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+use Illuminate\Broadcasting\PresenceChannel;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
+
+class UserNotification implements ShouldBroadcast
+{
+    use Dispatchable, InteractsWithSockets, SerializesModels;
+
+    public $notification;
```
Comment (quynt-1571) [inline app/Events/UserNotification.php]: nên khai báo là `protected`

**Review by haonguyen4993 — APPROVED** [review]

**Review by belt-1782 — APPROVED** [review]

File: laravel-echo-server.lock [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+	"process": 29894
+}
```
Comment (quanttn-2024) [inline laravel-echo-server.lock]: file này để làm gì anh nhỉ? thiếu end line nè anh

File: app/Http/Controllers/Admin/Group/GroupMemberController.php [hunk 4]
```diff
@@ -97,6 +106,9 @@ public function destroy(Group $group, Request $request, $userId)
         $user = $group->users()->detach($userId);
         if ($user)
         {
```
Comment (tanpv-2025) [inline app/Http/Controllers/Admin/Group/GroupMemberController.php]: convention anh ơi 
```diff
-if ($user)
-{
+if ($user) {
```

**Review by leductienttkt — APPROVED** [review]

**Pull request merged**